src/display/editor/alt_text.js

```diff
@@ -75,6 +75,16 @@ class AltText {
     const onClick = event => {
       event.preventDefault();
       this.#editor._uiManager.editAltText(this.#editor);
+      if (this.#useNewAltTextFlow) {
+        const label =
+          (this.#altText && "added") ||
+          (this.#altText === null && this.guessedText && "review") ||
+          "missing";
```

Comment:
This appears to be duplicating the code in https://github.com/mozilla/pdf.js/blob/b6b99a7b75e9aeab0fd47bfb9a3d4e37f994f970/src/display/editor/alt_text.js#L217-L224, so can we please move that code/comment to a new (private) helper method and use that in both places instead?